js/components/ChildInfoForm/ChildInfoForm.js

```diff
@@ -0,0 +1,110 @@
+import React, { Fragment } from 'react';
+import { TextInput, Text, View } from 'react-native';
+import { Form, Field } from 'react-final-form';
+import styles from './styles';
+import PropTypes from 'prop-types';
+
+const ChildInfoForm = ({ updateFormValues, child }) => {
+  console.log(child);
+  return (
+    <Form
+      onSubmit={() => {}}
+      validate={() => {}}
```

Comment:
Is this for later?

Reply:
I removed validate, however even though I do not need onSubmit it is a required prop